humanitec-resource-defs/azure-blob/basic/main.tf

```diff
@@ -1,7 +1,27 @@
+locals {
+  def_id         = "${var.prefix}azure-blob-storage-basic"
+  remote_backend = <<EOT
+terraform {
+  required_version = ">= 1.0.0"
+
+  %{if var.terraform_state != null}backend "azurerm" {
+    subscription_id       = "${var.terraform_state.subscription_id}"
```

Comment:
Stupid question: does this work? As I hit several times the error related to backend that can't be configured dynamically (so using vars).

Reply:
Yes, but mainly because the template is already fully rendered here (the variables are resolves), so the actual TF run only has hardcoded values.

Reply:
Ok, I got it. Thank you.